src/main/kotlin/com/vanniktech/dependency/graph/generator/DependencyGraphGeneratorPlugin.kt

```diff
@@ -20,10 +21,15 @@ open class DependencyGraphGeneratorPlugin : Plugin<Project> {
     extension.projectGenerators.all { projectGenerator ->
       project.tasks.register(projectGenerator.gradleTaskName, ProjectDependencyGraphGeneratorTask::class.java) {
         it.projectGenerator = projectGenerator
-        it.group = "reporting"
+        it.group = group
         it.description = "Generates a project dependency graph${projectGenerator.name.nonEmptyPrepend(" for ")}"
         it.outputDirectory = File(project.buildDir, "reports/project-dependency-graph/")
       }
+      project.tasks.register(projectGenerator.projectDependenciesTaskName, ProjectDependenciesTask::class.java) {
```

Comment:
What's the difference to the task above? Can you share an image?

Reply:
Being honest, it's been super long since I was active in that code. Forgot most of it

Reply:
I think this is for registering a new task `projectDependencies`. This task will list all the project dependencies.

Reply:
Yes and what's the difference to the already existing `generateProjectDependencyGraph` task?

Reply:
It will provide the information to display when the user run `. /gradlew tasks`.

Reply:
./gradlew tasks
<img width="689" alt="Screen Shot 2021-10-01 at 8 42 44 am" src="https://user-images.githubusercontent.com/1089679/135539614-fce2f42c-6200-4777-884c-1bb504e0dba7.png">
>

./gradlew app:projectDependencies
<img width="303" alt="Screen Shot 2021-10-01 at 8 43 23 am" src="https://user-images.githubusercontent.com/1089679/135539606-c4f7618f-06b0-40bf-8cb2-15d077dbcb83.png">

Reply:
Isn't this a bit out of scope for this plugin?

Reply:
Can we think it is the plain text version of the ProjectDependencyGraph?

Reply:
If there would be some kind of graph yes, but right now it's only a list and does not exactly fit this plugin

Reply:
Cool. No problem. I will close this merge request.